memtier_benchmark.cpp

```diff
@@ -1070,9 +1070,29 @@ int main(int argc, char *argv[])
             fprintf(stderr, "error: use no-expiry only with data-import\n");
             exit(1);
         }
-        
         obj_gen = new object_generator();
         assert(obj_gen != NULL);
+    }
+    else if (cfg.crc_verify) {
+        if (cfg.data_verify) {
+            fprintf(stderr, "error: use data-verify only with data-import\n");
```

Comment:
can you please explain the error message?

Reply:
It's a parameter of the memtie's data verify, I don't want to mix between the two machanisms